### DISCUSSION TITLE
How to connect to Slack using an email?

### DISCUSSION THREAD

**Discussion by inpefess** [Lightning-AI/torchmetrics#814] [Q&A] [answered]
How can I log in to https://join.slack.com/t/pytorch-lightning/shared_invite/zt-pw5v393p-qRaDgEk24~EjiZNBpSQFgQ without Google/Apple account using another email domain (like hotmail/yahoo etc)?

**Comment by Borda** [comment, marked as answer]
We are sorry for the confusion, seems that the link expired, let us share the actual one: https://join.slack.com/t/pytorch-lightning/shared_invite/zt-12iz3cds1-uyyyBYJLiaL2bqVmMN7n~A
